specifyweb/frontend/js_src/lib/components/QueryBuilder/Fields.tsx

```diff
@@ -155,11 +155,11 @@ export function QueryFields({
         fieldsContainerRef.current.clientHeight !==
           fieldsContainerRef.current.scrollHeight &&
         fields.length > oldFieldCount.current
-      )
-        scrollIntoView(
-          fieldsContainerRef.current.lastChild as HTMLElement,
-          'nearest'
-        );
+      ) {
+        const lastElement = fieldsContainerRef.current.lastChild as HTMLElement;
+        const firstNonContentsChild = lastElement.querySelector('button');
```

Comment:
`firstNonContentsChild` - great variable naming

I am still learning this, but yeah, when you have a descriptive variable, that removes the need for a comment, that's the best (because comment can get lost or out of date, so best code is code that is self-descriptive)